protected/humhub/modules/installer/forms/LocalisationForm.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\installer\forms;
+
+use humhub\libs\DynamicConfig;
+use humhub\libs\TimezoneHelper;
+use Yii;
+use yii\base\Model;
+
+/**
+ * LocalisationForm
+ *
+ * @since 1.17
+ */
+class LocalisationForm extends Model
+{
+    public ?string $language = null;
+    public ?string $timeZone = null;
+
+    /**
+     * @inheritdoc
+     */
+    public function init()
+    {
+        parent::init();
+
+        $this->language = Yii::$app->settings->get('defaultLanguage');
+        $this->timeZone = Yii::$app->settings->get('serverTimeZone');
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function rules()
+    {
+        return [
+            [['language'], $this->hasLanguages() ? 'required' : 'safe'],
+            [['timeZone'], 'required'],
+            [['language'], 'in', 'range' => array_keys($this->getLanguageOptions())],
+            [['timeZone'], 'in', 'range' => array_keys($this->getTimeZoneOptions())],
+        ];
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function attributeLabels()
+    {
+        return [
+            'language' => Yii::t('InstallerModule.base', 'Default Language'),
+            'timeZone' => Yii::t('InstallerModule.base', 'Default Timezone'),
+        ];
+    }
+
+    public function save(): bool
+    {
+        if (!$this->validate()) {
+            return false;
+        }
+
+        Yii::$app->db;
```

Comment:
Is this required?

Reply:
https://github.com/humhub/humhub/pull/7243/commits/8a30aae0e95545692516daf5c5f52cb0851ca2d7#diff-aa96ce8679a1100f0a92ba7e429ea682632777bb866fb499d5ab941446b68af3L66 - no, we don't need, I think it was copied by mistake, thanks.